lib/siwapp_web/controllers/api/recurring_invoices_controller.ex

```diff
@@ -0,0 +1,40 @@
+defmodule SiwappWeb.RecurringInvoicesController do
```

Comment:
You should name these API modules like `SiwappWeb.Api.RecurringInvoicesController` and the same with the InvoicesController. This way we'll know they are API controllers and not HTML controllers.